com.civicrm.hrjobroles/js/controllers/HRJobRolesController.js

```diff
@@ -330,10 +328,36 @@ define(['controllers/controllers'], function(controllers){
             $scope.checkNewRole = function checkNewRole() {
 
                 return (typeof $scope.edit_data['new_role_id'] === 'undefined'
-                    || typeof $scope.edit_data['new_role_id']['title'] === 'undefined'
-                    || $scope.edit_data['new_role_id']['title'] == ''
-                    || typeof $scope.edit_data['new_role_id']['job_contract_id'] === 'undefined'
-                    || $scope.edit_data['new_role_id']['job_contract_id'] == '');
+                || typeof $scope.edit_data['new_role_id']['title'] === 'undefined'
+                || $scope.edit_data['new_role_id']['title'] == ''
+                || typeof $scope.edit_data['new_role_id']['job_contract_id'] === 'undefined'
+                || $scope.edit_data['new_role_id']['job_contract_id'] == '');
+            };
+
+            /**
+             * @description Parse dates so they can be correctly read by server.
```

Comment:
You can omit the `@description` tag here, also make sure that you have a new line between the description and the rest of the tags
```js
/**
 * Lorem ipsum dolor sit amet
 *
 * @param {type} param1
 * @return {type}
 */